### ISSUE TITLE
[Question]: Installed everything. Don't know how to use anything

### ISSUE THREAD

**Issue by YorkyPoo** [danielmiessler/fabric#287]
### What is your question?

I've installed fabric. I added my OpenAI, Claude, YT, keys. I Installed Ollama. PowerShell can't find it. 
1. Do I need to add "C:\Users\slyre\AppData\Local\Ollama" to the system variables path?
2. Chatting with a model says I've passed my quota. Do I need to sign up for a subscription to use GPT-3.5 or Claude as a model?
3. I have Docker running but what do I need to do with Ollama and that?
4. My framework is in "C:\Program Files\Code\fabric", how do I use that with Ollama? I believe I have to pick the Ollama model somehow.
5. Do I need to be in the fabric folder when using Ollama for these commands?
6. Where can I see a list of viable commands and can I make a hot key for them? I see people using "pbpaste" but I'm guessing that's only for Mac and "Set-Clipboard" is for PowerShell.
7. Why do I have to type "fabric" before most commands and other people don't?
8. Do I need to install the Nvidia container toolkit for Docker to use the GPU?

**Comment by grayerbeard** [comment]
Are you stilll waiting for an answer or have you given up?
Which OS are you On?

**Comment by YorkyPoo** [comment]
I got it up and working. I'm on Windows. I had to use ChatGPT to figure out I have to format my commands like: "Get-Clipboard | fabric --model mistral:latest --pattern extract_wisdom". I've started using Ollama WebUI and Anything LLM trying to get my LLM to run on my home PC and I access it on my phone and Laptop. Problems I am having with that are the WebUI will occasionally drop all the models and I have to figure out how to reset that, Docker, and Ngrok. Other than that I am just trying to find out how to make one Into a widget to add to my free Study website.
